### ISSUE TITLE
[Bug]: Deployment on digital ocean fails

### ISSUE THREAD

**Issue by LukasGerm** [remix-run/remix#971]
### Which Remix packages are impacted?

- [X] `remix` (Remix core)
- [X] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [X] `@remix-run/dev`
- [ ] `@remix-run/express`
- [ ] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [X] `@remix-run/react`
- [X] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

1.0.6

### What version of Node are you using? Minimum supported version is 14.

I assume above 14

### Steps to Reproduce

Create a remix app, push it to a public repository and create a new app at digital ocean cloud.

### Expected Behavior

Deployment succeeds and application is available online.

### Actual Behavior

Some crazy NPM errors

```
[lukasgermerott-de] [2021-12-09 15:06:41] npm WARN old lockfile 
[lukasgermerott-de] [2021-12-09 15:06:59] 
[lukasgermerott-de] [2021-12-09 15:06:59] > postinstall
[lukasgermerott-de] [2021-12-09 15:06:59] > remix setup node
[lukasgermerott-de] [2021-12-09 15:06:59] 
[lukasgermerott-de] [2021-12-09 15:07:00] /usr/bin/env: 'node ': No such file or directory
[lukasgermerott-de] [2021-12-09 15:07:00] npm notice 
[lukasgermerott-de] [2021-12-09 15:07:00] npm notice New minor version of npm available! 8.1.4 -> 8.2.0
[lukasgermerott-de] [2021-12-09 15:07:00] npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.2.0>
[lukasgermerott-de] [2021-12-09 15:07:00] npm notice Run `npm install -g npm@8.2.0` to update!
[lukasgermerott-de] [2021-12-09 15:07:00] npm notice 
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR! code 127
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR! path /workspace
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR! command failed
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR! command sh -c remix setup node
[lukasgermerott-de] [2021-12-09 15:07:00] 
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR! A complete log of this run can be found in:
[lukasgermerott-de] [2021-12-09 15:07:00] npm ERR!     /home/apps/.npm/_logs/2021-12-09T15_07_00_080Z-debug.log
[lukasgermerott-de] [2021-12-09 15:07:00] 
[lukasgermerott-de] [2021-12-09 15:07:00] unable to invoke layer creator
[lukasgermerott-de] [2021-12-09 15:07:00] installing node_modules: exit status 127
[lukasgermerott-de] [2021-12-09 15:07:00] ERROR: failed to build: exit status 1
[lukasgermerott-de] [2021-12-09 15:07:01] 
[lukasgermerott-de] [2021-12-09 15:07:01] For documentation on the buildpacks used to build your app, please see:
[lukasgermerott-de] [2021-12-09 15:07:01]    Node.js: https://do.co/apps-buildpack-node
[lukasgermerott-de] [2021-12-09 15:07:01] 
[lukasgermerott-de] [2021-12-09 15:07:01]  ! Build failed (145)
```

This is the appspec

```

alerts:
- rule: DEPLOYMENT_FAILED
- rule: DOMAIN_FAILED
name: verynicewebsite
region: fra
services:
- build_command: npm run build
  environment_slug: node-js
  github:
    branch: main
    deploy_on_push: true
    repo: [repourl]
  http_port: 3000
  instance_count: 1
  instance_size_slug: basic-xxs
  name: lukasgermerott-de
  routes:
  - path: /
  run_command: npm start
  source_dir: /

```

**Comment by mcansh** [comment]
we recently merged a fix for this to dev #849, we're planning on a patch release tomorrow with this included

**Comment by LukasGerm** [comment]
Thanks alright :)
